test/playwright/utils/navigation.ts

```diff
@@ -248,7 +248,7 @@ export const goToSearchTerm = async (
       page.waitForNavigation(),
       page.click(`[aria-label="${t('search.search', dir)}"]`),
     ])
-    await page.waitForLoadState('networkidle')
+    await page.waitForLoadState('domcontentloaded')
```

Comment:
This makes the tests faster.

Reply:
Much faster indeed. I'm not sure 100% we should use `domcontentloaded` over `load` because `load` also downloads stylesheets and images.

Reply:
You are right, we should use `load` to make sure that all the images are there for the snapshots. Thank you!